src/DynamoCoreWpf/ViewModels/Search/NodeAutoCompleteSearchViewModel.cs

```diff
@@ -75,9 +84,16 @@ internal IEnumerable<NodeSearchElement> GetMatchingNodes()
 
             foreach (var descriptor in functionDescriptors)
             {
-                if (descriptor.ReturnType.ToString() == inputPortType)
+                if ((descriptor.ReturnType.ToString() == inputPortType) || DerivesFrom(inputPortType, descriptor.ReturnType.ToString(), dynamoViewModel.Model.EngineController.LiveRunnerCore))
```

Comment:
Is the `LiveRunnerCore` available in manual mode?

Reply:
yes I believe so, the `liveRunnerServices` and `core` constructed when the engineController is constructed - that should definitely be around in manual mode.

also, my manual testing performs fine in manual mode.

Reply:
Code block node autocomplete uses the `LibraryManagementCore` on `LibraryServices`, which is the static core. I've noticed that we have existing issues due to these cores not matching up exactly but that is a separate VM discussion I guess. I'm okay with using the LiveRunnerCore, just that we should be cognizant of such issues.

Reply:
hmm, let me take a look at using that core if you think it will be more stable? Which one is more "up to date" ?